pkg/controller/service/service_controller.go

```diff
@@ -248,7 +248,12 @@ func (s *ServiceController) processServiceUpdate(cachedService *cachedService, s
 	cachedService.state = service
 	err, retry := s.createLoadBalancerIfNeeded(key, service)
 	if err != nil {
+		eventType := "CreatingLoadBalancerFailed"
 		message := "Error creating load balancer"
+		if !wantsLoadBalancer(service) {
+			eventType = "ProcessingOrphanedLoadBalancerFailed"
```

Comment:
Humm, calling this "orphaned" sounds like something goes wrong in controller even before this error. Probably just name it something normal? `DeleteLoadBalancerFailed` or `CleanupLoadBalancerFailed`?